scripts/exgfs_wave_post_gridded_sbs.sh

```diff
@@ -55,7 +55,7 @@
   echo "Starting at : $(date)"
   echo '-------------'
   echo ' '
-  [[ "$LOUD" = YES ]] && set -x
+  ${TRACE_ON:-set -x}
```

Comment:
Technically this is correct, but the entire `set +x` before and echo and `set -x` after the echo is useless when `TRACE_ON` is set.
What real difference does it make to do `set +x` for an echo statement when the default project wide is `set -x`?

Reply:
I didn't want to go through all the (mostly wave) scripts and change this overall paradigm just yet (this PR is big enough). When I move on to giving these scripts closer attention, I plan on revisiting this. Simplest solution then will be to leave trace on (if it is on) and echo a heredoc instead of individual echo lines (which is how all multiline echos should be handled).

Reply:
Understood.